dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionGroupRestControllerIT.java

```diff
@@ -0,0 +1,2094 @@
+                        .andExpect(status().isNoContent());
+    }
+
+    // This is currently not supported in DSpace API
+    @Ignore
+    @Test
+    public void deleteCollectionDefaultBitstreamReadGroupTestCollectionAdmin() throws Exception {
+
+        String bitstreamGroupString = "BITSTREAM";
+        int defaultBitstreamRead = Constants.DEFAULT_BITSTREAM_READ;
+
+        Group role = createDefaultReadGroup(bitstreamGroupString, defaultBitstreamRead);
+        authorizeService.addPolicy(context, collection, Constants.ADMIN, eperson);
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/bitstreamReadGroup"))
+                        .andExpect(status().isNoContent());
+
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/bitstreamReadGroup"))
+                        .andExpect(status().isNoContent());
+    }
+
+    @Test
+    public void deleteCollectionDefaultBitstreamReadGroupUnAuthorizedTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        String bitstreamGroupString = "BITSTREAM";
+        int defaultBitstreamRead = Constants.DEFAULT_BITSTREAM_READ;
+
+        Group role = createDefaultReadGroup(bitstreamGroupString, defaultBitstreamRead);
+        context.restoreAuthSystemState();
+
+        getClient().perform(delete("/api/core/collections/" + collection.getID() + "/bitstreamReadGroup"))
+                   .andExpect(status().isUnauthorized());
+    }
+
+    @Test
+    public void deleteCollectionDefaultBitstreamReadGroupForbiddenTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        String bitstreamGroupString = "BITSTREAM";
+        int defaultBitstreamRead = Constants.DEFAULT_BITSTREAM_READ;
+
+        Group role = createDefaultReadGroup(bitstreamGroupString, defaultBitstreamRead);
+        context.restoreAuthSystemState();
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/bitstreamReadGroup"))
+                        .andExpect(status().isForbidden());
+    }
+
+
+    @Test
+    public void deleteCollectionDefaultBitstreamReadGroupNotFoundTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        String bitstreamGroupString = "BITSTREAM";
+        int defaultBitstreamRead = Constants.DEFAULT_BITSTREAM_READ;
+
+        Group role = createDefaultReadGroup(bitstreamGroupString, defaultBitstreamRead);
+        context.restoreAuthSystemState();
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        getClient(token).perform(delete("/api/core/collections/" + UUID.randomUUID() + "/bitstreamReadGroup"))
+                        .andExpect(status().isNotFound());
+    }
+
+    @Test
+    public void getWorkflowGroupForCollectionAndRole() throws Exception {
```

Comment:
please apply the suggestion made for the other groups also to the workflow tests as I found the same approach / area of improvements

Reply:
Done.